src/nvim/os/fs.c

```diff
@@ -192,13 +192,15 @@ int os_open(const char* path, int flags, int mode)
 
 /// Get stat information for a file.
 ///
-/// @return OK on success, FAIL if a failure occurred.
+/// @return `true` on success, `false otherwise.
 static bool os_stat(const char *name, uv_stat_t *statbuf)
   FUNC_ATTR_NONNULL_ALL
 {
   uv_fs_t request;
```

Comment:
That works.

When [LibUV initializes the `uv_fs_t` data structure](https://github.com/libuv/libuv/blob/v1.x/src/unix/fs.c#L63-L70), it doesn't assign the `statbuf` field. So I guess it is a LibUV bug?